db/migrate/20170427152006_remove_invalid_hawkular_endpoints.rb

```diff
@@ -0,0 +1,32 @@
+class RemoveInvalidHawkularEndpoints < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :endpoints, :as => :resource, :dependent => :destroy, :autosave => true
+  end
+
+  class Endpoint < ActiveRecord::Base; end
+
+  def up
+    Endpoint.where(
+      :resource_type => "ExtManagementSystem",
+      :role          => 'hawkular',
```

Comment:
are we sure say middleware providers don't use this role?
perhaps constrain EMS types like you did in `down`.